### PR TITLE
Menu branch

### PR DESCRIPTION
**Pull request by Metsanpeitto**
In this project I needed to implement the following features for the mobile version : 
-When the user clicks (or taps) the hamburger button, the mobile menu appears.
-When the user clicks (or taps) the close (X) button, the mobile menu disappears.
-When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,11 +46,28 @@
...
+          <div class="line-3"></div>
+        </div>
+
         <div class="menu-buttons">
-          <a href="/" class="menu-button">Portfolio</a>
-          <a href="/" class="menu-button">About</a>
-          <a href="/" class="menu-button">Contact</a>
+          <a href="#landing" class="menu-button">Portfolio</a>
+          <a href="#about" class="menu-button">About</a>
+          <a href="#contact" class="menu-button">Contact</a>
```
Comment (Davidosky007) [inline index.html]: * when i click on the mobile menu option the mobile menu should disappear as stated in the project requirement, please make the necessary adjustment

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @Metsanpeitto @ntzwilly, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!
CR3_gif_almost_there
# Requested Changes ♻️ 
* well done on your work so far 👍 but there are still some changes to be made please check the comments below 🙏 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Metsanpeitto) [reply index.html]: Thanks, Davidosky007. I missed out on that detail.
Then I realized that the menu buttons were lacking some features. So I added some js code to make them close the modal when the button is clicked and the page automatically scrolls to the section desired.
For the desktop version, I added some more js code, so now, when the user scrolls over the different sections, the button corresponding to that section gets underlined.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**